tests/unit/Dashboard.spec.ts

```diff
@@ -34,7 +34,23 @@ describe('Dashboard - UI', () => {
 
     // create wrapper for Dashboard
     // this stubs out the 5 sub-components
-    wrapper = shallowMount(Dashboard, { store, vuetify, mocks: { $route } })
+    wrapper = shallowMount(Dashboard, { store,
+      vuetify,
+      mocks: { $route },
+      data: () => ({
+        isInGoodStanding: true
+      }),
+      computed: {
+        isGoodStanding: {
+          get (): boolean {
+            return this.$data.isInGoodStanding
+          },
+          set (val: boolean) {
+            this.$data.isInGoodStanding = val
```

Comment:
Setting local data in the wrapper allows us to mock getters in jest. isGoodStanding in Dashboard is overridden and and the wrapper will rely on the local computed version. This allows us to change the getter for tests as done below.